src/common/components/rc-info/index.tsx

```diff
@@ -0,0 +1,207 @@
+import React, { useEffect, useState } from 'react'
+import { Button, Modal } from "react-bootstrap";
+import { _t } from "../../i18n";
+import { findRcAccounts } from "../../api/hive";
+import { ResourceCreditsDelegation } from "../rc-delegation"
+import { RcDelegationsList } from '../rc-delegations-list';
+
+export const ResourceCreditsInfo = (props: any) => {
+
+    const [showRcInfo, setShowRcInfo] = useState(false);
+    const [delegated, setDelegated] = useState();
+    const [receivedDelegation, setReceivedDelegation] = useState();
+    const [resourceCredit, setresourceCredit] = useState();
+    const [showDelegationModal, setShowDelegationModal] = useState(false)
+    const [showDelegationsList, setShowDelegationsList] = useState(false)
+    const [listMode, setListMode] = useState("")
+    const [toFromList, setToFromList] = useState('')
+    const [amountFromList, setAmountFromList]: any = useState('')
+
+    const { rcPercent, account, activeUser } = props;
+
+    useEffect(() => {
+        findRcAccounts(account.name).then((r) => {
+          console.log(r)
+            const outGoing = r.map((a: any) => a.delegated_rc);
+            const delegated = outGoing[0]
+            const formatOutGoing: any =  rcFormatter(delegated)
+            setDelegated(formatOutGoing);
+            const availableResourceCredit: any = r.map((a: any) => a.rc_manabar.current_mana);
+            console.log(availableResourceCredit)
+            // const formatRc: any = rcFormatter(Number(availableResourceCredit));
+            setresourceCredit(availableResourceCredit);
+            const inComing: any = r.map((a: any) => Number(a.received_delegated_rc));
+            const formatIncoming = rcFormatter(inComing);
+            setReceivedDelegation(formatIncoming);
+          })
+    }, []);
+
+    const rcFormatter = (num: number) => {
```

Comment:
@Adesojisouljay I have moved this function to helpers, please merge development branch into this and use function from helpers. Reference: https://github.com/ecency/ecency-vision/pull/1194